deployment/kubernetes/flood-api.yaml

```diff
@@ -78,24 +78,20 @@ spec:
   selector:
     app: flood-frontend
 ---
-apiVersion: apisix.apache.org/v2
-kind: ApisixRoute
+apiVersion: traefik.io/v1alpha1
+kind: IngressRoute
 metadata:
   name: flood-frontend
 spec:
-  http:
-    - name: flood-frontend
-      match:
-        paths:
-          - /*
-        hosts:
-          - flood-dev1.openepi.io
-          - flood-dev2.openepi.io
-          - flood-test.openepi.io
-          - flood.openepi.io
-      backends:
-        - serviceName: flood-frontend
-          servicePort: 80
+  entryPoints:
+    - web
+  routes:
+  - kind: Rule
+    match: Host(`flood-dev1.openepi.io`) || Host(`flood-dev2.openepi.io`) || Host(`flood-test.openepi.io`) || Host(`flood.openepi.io`)
```

Comment:
Did you check if it is possible to use a wildcard here? It wasn't possible with APISIX, but maybe traefik supports it

Reply:
I was just lazy, and did the simplest thing I could find :smiling_face_with_tear: 
They support HostRegexp which will do the trick. Will change.